tests/_envs/chrome-driver.yml

```diff
@@ -1,3 +1,9 @@
+extensions:
+  enabled:
+    # Run ChromeDriver in the background when acceptance tests are running.
+    - Codeception\Extension\RunProcess:
+        0: ./build/tmp/webdriver/chromedriver --url-base=/wd/hub
```

Comment:
We can probably wait on #7433 and #7345 before merging this.

#7433 removes the `chrome-driver` environment in favor of a `custom` environment, so it'll change how this should work.

And then #7345 updates the chromedriver install task which makes it easier to get the latest version, which this PR currently assumes you'll have.